src/Symfony/Bridge/Doctrine/SchemaListener/AbstractSchemaListener.php

```diff
@@ -28,32 +31,48 @@ protected function getIsSameDatabaseChecker(Connection $connection): \Closure
     {
         return static function (\Closure $exec) use ($connection): bool {
             $schemaManager = method_exists($connection, 'createSchemaManager') ? $connection->createSchemaManager() : $connection->getSchemaManager();
-            $checkTable = 'schema_subscriber_check_'.bin2hex(random_bytes(7));
+            $checkTable = 'schema_subscriber_check';
```

Comment:
```suggestion
            $checkTable = '_schema_subscriber_check';
```

Having a `_` prefix is kind of a convention to signal this table is "internal" and to avoid name clashes (even if very unprobable).